www/klaj01/homework-08/js/script.js

```diff
@@ -54,7 +54,7 @@ App.fetchRepositories = function (username) {
   }).done(function (repositories) {
     $('<div class="loader"></div>').remove()
     repositories.forEach(function (repository) {
-      html += `
+      html = `
```

Comment:
špatně, uvědom si, že do této proměnné dokola jen přiřazuješ nějakou novou hodnotu, což nakonec v aplikaci znamená, že zobrazuješ jen poslední repozitář, ne všechny.
např.
```
var html = '';
var animals = ['dog', 'cat', 'mouse'];
animals.forEach(function (a) {
    html = a;
});
console.log(html);
```
Co se vypíše na konzoli?

Měl bys svou aplikaci před odevzdáním nejprve otestovat, v zadání je na to také požadavek.

Reply:
Chápu v čem je chyba, ale nemám moc času se tomu věnovat, spíš bych chtěl už udělat pull request na svojí appku.  Jestli to teda nevadí, tak prosím o uzavření tohoto úkolu. Děkuju

Reply:
Chyba byla v tom ze jsi neinicializoval promennou `html`, to bylo vse.

Reply:
Chápu, děkuju. Příště na to dám pozor